### ISSUE TITLE
Can you support the Uint8List type ? Thanks.

### ISSUE THREAD

**Issue by lbx-ai** [synw/sqlcool#5]

**Comment by synw** [comment]
Hi. I added a `blob` method to schema constructor: use it like for other types:

```
DbTable("mytable")..blob("myblob")
```
Please check if everything works ok, using this in your pubspec:

```
sqlcool:
    git:
        url: git://github.com/synw/sqlcool.git
        ref: master
```
I'll release it if everything works as expected

**Comment by synw** [comment]
I published version 2.9.0 because some fixes where needed. This feature is included, let me know if anything goes wrong
